### ISSUE TITLE
Conflict between SA1003 and SA1019 for '++.'

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#2471]
**Input:**

```
_connectionCount++.ToString();
```

**Actual result:**

SA1003 is reported, and the code fix produces the following:

```
_connectionCount++ .ToString();
```

After the code fix, SA1019 is reported, and the code fix restores the original form.

**Expected result:**

One of the two forms above should not produce SA1003 or SA1019. Based on the behavior of the Visual Studio code formatter, it appears that the preferred form (i.e. neither of these warnings shown) should be the original input above. When implementing handling for this situation, make sure to also handle the `?.` (part of SA1019) and `->` (part of SA1023) operators.
